src/model/Term.ts

```diff
@@ -180,12 +172,14 @@ export default class Term extends Asset implements TermData {
                 return res;
             });
         }
-        if (withoutExacts) {
-            delete result.exactMatches;
-        } else if (result.exactMatches) {
-            result.exactMatches = result.exactMatches.map((pt: Term) =>
-                pt.toTermData(true)
-            );
+        if (result.exactMatchTerms) {
+            result.exactMatchTerms = result.exactMatchTerms.map( (em : Term) =>
+            { return {
+                types: em.types,
+                iri : em.iri,
+                label: em.label,
+                vocabulary: em.vocabulary
+            }}) ;
```

Comment:
`Term.exactMatchTerms` are (or at least should be based on the class declaration) instances of `TermInfo`, not `Term`, so I don't see any reason for this explicit processing.